pmda/test/test_rdf_s.py

```diff
@@ -0,0 +1,119 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# PMDA
+# Copyright (c) 2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+from __future__ import absolute_import, division
+
+import pytest
+
+from numpy.testing import assert_almost_equal
+
+import MDAnalysis as mda
+import numpy as np
+from pmda.rdf import InterRDF_s
+from MDAnalysis.analysis import rdf
+
+from MDAnalysisTests.datafiles import GRO_MEMPROT, XTC_MEMPROT
+
+
+@pytest.fixture(scope='module')
+def u():
+    return mda.Universe(GRO_MEMPROT, XTC_MEMPROT)
+
+
+@pytest.fixture(scope='module')
+def sels(u):
+    s1 = u.select_atoms('name ZND and resid 289')
+    s2 = u.select_atoms(
+         '(name OD1 or name OD2) and resid 51 and sphzone 5.0 (resid 289)')
+    s3 = u.select_atoms('name ZND and (resid 291 or resid 292)')
+    s4 = u.select_atoms('(name OD1 or name OD2) and sphzone 5.0 (resid 291)')
+    ags = [[s1, s2], [s3, s4]]
+    return ags
+
+
+@pytest.fixture(scope='module')
+def rdf_s(u, sels):
```

Comment:
Test with different schedulers:
```python
def rdf_s(u, sels, scheduler)
```

Reply:
(The `scheduler` fixture is defined in `conftest.py` and should run this calculation with all dask schedulers.)